app/views/vm_common/_reconfigure.html.haml

```diff
@@ -327,11 +327,12 @@
           %td
             {{networkAdapter.name}}
           %td{"ng-if" => "!vm.isVmwareCloud() && networkAdapter.add_remove == 'update'"}
-            - default_option = [("{{networkAdapter.vlan}}"),'']
+            - default_item = "{{networkAdapter.vlan}}"
+            - vlan_options = @vlan_options.blank? ? [default_item] : @vlan_options.uniq
```

Comment:
This seems like hacking around the problem, since the duplicates are still around somewhere before this code.